src/header/header.scss

```diff
@@ -429,3 +386,20 @@
     }
   }
 }
+// #endregion Header buttons
+
+// #region Header extras
+@mixin iui-header-divider {
+  width: 1px;
+  height: $iui-baseline * 2;
+  @include themed {
+    background-color: t(iui-color-background-4);
+  }
+}
+
+@mixin iui-header-chevron {
+  @include iui-icons-small;
```

Comment:
maybe worth including this mixin for all icons in header? it adds flex display, not sure its gonna be better or worse..